packages/api-derive/src/balances/all.ts

```diff
@@ -61,9 +62,11 @@ function calcBalances (api: ApiInterfaceRx, [{ accountId, accountNonce, freeBala
     isVesting,
     lockedBalance,
     lockedBreakdown,
+    perBlock,
```

Comment:
```suggestion
    vestingPerBlock,
```

(And then just alphabetical)

The rest looks fine.

Reply:
done